tools/site_and_regional/run_neon.py

```diff
@@ -534,6 +535,9 @@ def run_case(
                     elif not setup_only:
                         print("Resubmitting case {}".format(case_root))
                         case.submit(no_batch=no_batch)
+                        logger.info("-----------------------------------")
+                        logger.info("Successfully submitted case!")
+                        logger.info("Use 'qstat -u <user-name>' to check its run status")
```

Comment:
This is really helpful to have the 'qstat' suggestion printed here. I think this will be a good way to inform users that things are running and what the best methods are to check on case status going forward.

Reply:
See my comment below...

Reply:
In the interest of getting this in before the flux workshop maybe we should just remove this line for now? I am not immediately sure how to call the xmlquery from this script since right now everything is done with the actual python objects.

but I imagine if we had that we could make some kind of function...


````
def get_queue_command(batch_system):
  match batch_system.lower():
      'pbs':
           queue_command = 'qstat -u <user_name>'
      'slurm':
           queue_command = 'squeue -u <user_name>'
      'cobalt':
          queue_command = 'qstat -u <user_name>'
      'lsf':
            queue_command = 'bqueues -u <user_name>'
      case _:
          queue_command = ''

    return queue_command
```

I think these are all correct...

Reply:
I'm wondering if another temporary fix option would be just to print something like "if using PBS, check case status with qstat -u user_name; if using slurm, check case status with ..."? Maybe it would look clunky, but it would at least inform users that it is useful to check output status. Just an idea.

Reply:
This was worked out...